packages/replay-next/components/console/filters/EventType.tsx

```diff
@@ -18,8 +18,14 @@ export default function EventType({
 }) {
   const { eventTypesForDisplay: eventTypes, update } = useContext(ConsoleFiltersContext);
 
-  const checked = eventTypes[event.type] === true;
-  const toggle = () => update({ eventTypes: { [event.type]: !checked } });
+  const checked = eventTypes[event.rawEventTypes[0]] === true;
```

Comment:
I wonder if there's an easier way to model this that doesn't involve toggling _all_ event types for a category?

Reply:
`rawEventTypes` is what the `runtime` and `backend` knows, and thus, that is what needs to be queried for. Usually, you won't have too many of these in one "normalized event type" though. It will be negligible for performance.

As we have previously discussed, the code for this ideally should at least be isolated and modularized. On the other hand, chromium should also not be crashing all the time, and is that latter part that is a bit more anxiety inducing for me for now 😅

What is your concern here in particular?

Reply:
I just meant there's maybe a simpler way to _model_ which categories are checked then adding an attribute to every event in the array.